config/services/galera.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Galera</short>
+  <description>MariaDB-Galera Database Server</description>
+  <port protocol="tcp" port="3306"/>
+  <port protocol="tcp" port="4567"/>
```

Comment:
Just noting that as per https://galeracluster.com/library/documentation/firewall-settings.html, this port can *optionally* be opened for multicast replication. I think it's okay to keep it closed until a user requests the UDP port.